resolver-dns/src/main/java/io/netty/resolver/dns/DnsResolveContext.java

```diff
@@ -213,12 +217,10 @@ public Throwable fillInStackTrace() {
         }
     }
 
-    private void doSearchDomainQuery(String hostname, FutureListener<List<T>> listener) {
+    void doSearchDomainQuery(String hostname, Promise<List<T>> nextPromise) {
         DnsResolveContext<T> nextContext = newResolverContext(parent, hostname, dnsClass, expectedTypes,
-                                                              additionals, nameServerAddrs);
-        Promise<List<T>> nextPromise = parent.executor().newPromise();
+                additionals, nameServerAddrs);
```

Comment:
nit: restore alignment?

Reply:
I also see a lot of similar issues in #8177. Perhaps we should make sure we all are using same IntelliJ code style settings?